docker-compose-services/drupalci-chromedriver/README.md

```diff
@@ -19,5 +20,10 @@ It is based on [Matt Glaman's excelent blog post](https://glamanate.com/blog/run
 user@demo-web:/var/www/html$ phpunit --verbose -c web/core/phpunit.xml.dist web/core/modules/system/tests/src/FunctionalJavascript/System/DateFormatTest.php
 ```
 
+## Notes
+
+* In order for this to work all other ddev instances must be turned off, only the Drupal core instance which will be used for the PHPUnit tests can be running.
```

Comment:
There's no reason this should be the case. What were you using for SIMPLETEST_BASE_URL?

Reply:
Oh, I see the problem. The chromedriver service needs 
```
links:
  web:web
```

The problem is described in https://github.com/drud/ddev/issues/2744 but it's an easy fix

Reply:
When I add "web:web" to the links list it gives this error:

`A service can not link to itself: web`

Reply:
Never mind, I added the link to the wrong service.

Reply:
I added the web:web link to the chromedriver services section so it now looks like this:

```
services:
  chromedriver:
    image:  drupalci/webdriver-chromedriver:production
    container_name: ddev-${DDEV_SITENAME}-chromedriver
    labels:
      com.ddev.site-name: ${DDEV_SITENAME}
      com.ddev.approot: $DDEV_APPROOT
    links:
      - web:web
```
However it now gives this error:
`Circular dependency between chromedriver and web and db `

Reply:
Thanks for exploring this so carefully. The issue is that "web" may be ambiguous, and "chromedriver" will be ambiguous if there are multiple projects running at the same time.  However, both mostly work without the links. But the addition of web in chromedriver will help. The one in web is less important. 

Here's what I recommend, https://gist.github.com/rfay/801d382cf08bebf4811b43ff0ea8fa31

This 
* changes the name of the service to "chrome", so we can use the hostname "chrome" instead of switching between "chromedriver" and "chrome"
* Uses "external links" for the chrome container to address <hostname>.ddev.site. This routes traffic through the router, which is probably a better test anyway. It *could* also point to "web" instead of ddev-router. 

I think this will probably do it. If it doesn't work right away for you, maybe we could work together a little bit in slack and get it nailed?